nrf_rpc/nrf_rpc.c

```diff
@@ -465,6 +467,27 @@ static int transport_init(nrf_rpc_tr_receive_handler_t receive_cb)
 	return err;
 }
 
+static int transport_init_all(nrf_rpc_tr_receive_handler_t receive_cb)
+{
+	void *iter;
+	const struct nrf_rpc_group *group;
+
+	for (NRF_RPC_AUTO_ARR_FOR(iter, group, &nrf_rpc_groups_array,
+				 const struct nrf_rpc_group)) {
+
+		transport_init_single(receive_cb, group);
```

Comment:
error is not checked

Reply:
This is intentional though, because I saw in the previous implementation they didn't return an error when a single initialization failed.
The logic here:
    https://github.com/nrfconnect/sdk-nrfxlib/blob/49299379e19fb3b2d7bc8cc1b0bd2803c93a6878/nrf_rpc/nrf_rpc.c#L420

Should be the same with the logic in the function transport_init_single basically. 

The error that I return in this function is the logic which exists here:
https://github.com/nrfconnect/sdk-nrfxlib/blob/49299379e19fb3b2d7bc8cc1b0bd2803c93a6878/nrf_rpc/nrf_rpc.c#L458